smf/cockroachdb/manifest.xml

```diff
@@ -16,12 +17,8 @@
       <envvar name="LD_LIBRARY_PATH" value="/opt/ooce/pgsql-13/lib/amd64" />
     </method_environment>
   </method_context>
-  <!--
-    TODO(https://github.com/oxidecomputer/omicron/issues/727)
-    Call "start", joining with other crdb instances
-  -->
   <exec_method type='method' name='start'
-    exec='ctrun -l child -o noorphan,regent /opt/oxide/cockroachdb/bin/cockroach start-single-node --insecure --listen-addr=%{config/listen_addr} --store=%{config/store} &amp;'
+    exec='/var/svc/manifest/site/cockroachdb/method_script.sh'
```

Comment:
Method scripts ought not to go into the manifest tree, I would put this at `/opt/oxide/lib/svc/manifest/cockroachdb.sh` instead.

Reply:
Moved